vendor/modules.txt

```diff
@@ -228,18 +234,29 @@ github.com/gogo/protobuf/proto
 github.com/gogo/protobuf/protoc-gen-gogo/descriptor
 github.com/gogo/protobuf/sortkeys
 github.com/gogo/protobuf/types
+# github.com/golang/groupcache v0.0.0-20191227052852-215e87163ea7
+github.com/golang/groupcache/lru
 # github.com/golang/protobuf v1.4.2 => github.com/golang/protobuf v1.3.5
 github.com/golang/protobuf/proto
 github.com/golang/protobuf/ptypes
 github.com/golang/protobuf/ptypes/any
 github.com/golang/protobuf/ptypes/duration
 github.com/golang/protobuf/ptypes/timestamp
+# github.com/google/crfs v0.0.0-20191108021818-71d77da419c9
+github.com/google/crfs/stargz
 # github.com/google/go-cmp v0.4.0
 github.com/google/go-cmp/cmp
 github.com/google/go-cmp/cmp/internal/diff
 github.com/google/go-cmp/cmp/internal/flags
 github.com/google/go-cmp/cmp/internal/function
 github.com/google/go-cmp/cmp/internal/value
+# github.com/google/go-containerregistry v0.0.0-20200425101607-48f605c3b60a
```

Comment:
I assume from this dependency that stargz makes its own registry connections (i guess expected) that are not compatible with what buildkit does through containerd libraries and I guess missing all the extra config+token logic? Not ideal

Reply:
Thanks for the feedback. Yes, the snapshotter makes its own registry connections and the logic isn't using containerd lib. I agree with that the logic should have compatibility with buildkit.

Reply:
opened https://github.com/containerd/stargz-snapshotter/issues/149 as a reminder